release-notes/3.0/preview/api-diff/preview4/standalone-packages/3.0-preview4-standalone-packages_System.IO.Pipelines.md

```diff
@@ -0,0 +1,28 @@
+# System.IO.Pipelines
+
+``` diff
+ namespace System.IO.Pipelines {
+     public class PipeOptions {
+-        public PipeOptions(MemoryPool<byte> pool = null, PipeScheduler readerScheduler = null, PipeScheduler writerScheduler = null, long pauseWriterThreshold = (long)32768, long resumeWriterThreshold = (long)16384, int minimumSegmentSize = 2048, bool useSynchronizationContext = true);
++        public PipeOptions(MemoryPool<byte> pool = null, PipeScheduler readerScheduler = null, PipeScheduler writerScheduler = null, long pauseWriterThreshold = (long)65536, long resumeWriterThreshold = (long)32768, int minimumSegmentSize = 4096, bool useSynchronizationContext = true);
```

Comment:
@davidfowl, is this considered a breaking change? And if so, presumably we are ok with this?

https://github.com/dotnet/corefx/pull/35939